iocore/net/SSLSNIConfig.h

```diff
@@ -60,12 +65,19 @@ struct PcreFreer {
 struct NamedElement {
   NamedElement() {}
 
+  NamedElement(NamedElement const &other)            = delete;
+  NamedElement &operator=(NamedElement const &other) = delete;
   NamedElement(NamedElement &&other);
   NamedElement &operator=(NamedElement &&other);
+  ~NamedElement() = default;
 
   void set_glob_name(std::string name);
   void set_regex_name(const std::string &regex_name);
 
+  inline static constexpr in_port_t MAX_PORT_VALUE{std::numeric_limits<in_port_t>::max()};
+  using port_range_t = swoc::DiscreteRange<in_port_t>;
```

Comment:
This shows up twice. Promote to "ts_ip.h"?

Reply:
Maybe I'll sleep on it. I don't think there's any harm in DRYing it up, but it's only in two places so my initial reaction was to wait until it happened in a third place.